pkg/bbgo/session.go

```diff
@@ -717,7 +737,17 @@ func (session *ExchangeSession) UpdatePrices(ctx context.Context, currencies []s
 	var lastTime time.Time
 	for k, v := range tickers {
 		// for {Crypto}/USDT markets
-		session.lastPrices[k] = v.Last
+		// map things like BTCUSDT = {price}
+		if market, ok := markets[k]; ok {
+			if types.IsFiatCurrency(market.BaseCurrency) {
+				session.lastPrices[k] = v.Last.Div(fixedpoint.One)
```

Comment:
What's the purpose of being divided by one?

Reply:
reversed quoting, e.g. USDT/TWD